pkg/operation/seed/seed.go

```diff
@@ -32,6 +37,8 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+const kibanaHostname = "k-seed"
```

Comment:
Why not just `k`?

Reply:
Currently checking if there will be any naming conflicts if they are both named just "k". For one seed we have two EFK stacks and consequently two kibanas that have to be exposed.